ons_alpha/jinja2/templates/components/streamfield/corrections_block.html

```diff
@@ -1,5 +1,10 @@
-{{ value.when|date("DATETIME_FORMAT") }}
-
-{{ value.text|richtext }}
-
-<a href="{{ routablepageurl(page, 'previous_version', version_num) }}">{{ _("View superseded version") }}</a>
+<div class="corrections__item">
+    <div class="corrections__meta">
+        <h2 class="ons-u-fs-r--b corrections__heading">Correction</h2>
+        <time class="corrections__date">{{ value.when|date("DATETIME_FORMAT") }}</time>
+    </div>
+    <div class="corrections__detail">
+        {{ value.text|richtext }}
```

Comment:
I'm not sure you need to use the `|richtext` filter when working with block values. One to check, I think

Reply:
I will test this without the filter to see if it makes a difference - it was certainly working using it.